test/unit/bin-tessel-2.js

```diff
@@ -285,7 +289,7 @@ exports['Tessel (cli: wifi)'] = {
     reject.catch(function() {
```

Comment:
@johnnyman727 Sorry! I was referring to these double catches over here, they are outside of the scope of the PR